Utilities/src/RoccoR.cc

```diff
@@ -0,0 +1,403 @@
+
```

Comment:
Can we have a comment here indicating where the code was checked out from? Good for traceability, especially if the code breaks in some future version, in which case we can see if the maintainers have updated it.

Reply:
it is a Twiki attachment, will comment in the link